docs/user-documentation/metadata.md

```diff
@@ -1,50 +1,180 @@
 
-Fields get mapped to RDF in (a yml file). See the RDF and Fedora section.
+## Vocabularies
 
-## Islandora demo
+In Drupal, _Taxonomy Vocabularies_ (or simply 'Vocabularies') are also entity subtypes that group fields and their configurations. Unlike content types, they are intended to be used as descriptive attributes of content and have hierarchy built in. Whereas instances of content types are called nodes, items in a vocabulary are called _terms_.
 
-TODO: document the fields and display modes in Islandora Demo.
+For example, Islandora includes the 'Islandora Models' vocabulary which includes the 'External URI' field. The 'Islandora Models' vocabulary includes the terms 'Audio', 'Binary', 'Collection', 'Image', and 'Video'. By linking to one of these terms in the 'Islandora Models' vocabulary a repository manager can tell that the node (digital object) should be considered an 'Image' or 'Audio', etcetera. The Controlled Access Terms module provides additional vocabularies representing Corporate Bodies, Persons, Families, Geographic Locations, and Subjects. Each of these vocabularies has its own set of fields allowing repositories to further describe them. Repository item nodes can then reference terms in these vocabularies. See 'Entity Reference fields' in the 'Field Types' section below.
 
-!!! tip "7.x Migration Note"
-    #### What about my XML?
-    In 7.x, metadata were (usually) stored within XML datastreams such as MODS or DC. In CLAW we suggest breaking out
-    individual metadata elements into fields attached to the node (or to an associated entity, if appropriate). The Metadata Interest Group is developing a default mapping which will provide a basic, yet customizable, transform between MODS metadata and Drupal fields in Islandora Demo.
+<!--[Geographic Location currently has hierarchy turned off. This section will apply once https://github.com/Islandora-CLAW/controlled_access_terms/pull/21 is merged.] Another example, this time illustrating hierarchy, is the 'Geographic Location' vocabulary. Although no terms are listed by default, a repository could create a 'Western Hemisphere' term and then create a 'North America' term as a child of the 'Western Hemisphere' term, etcetera.
 
-     It is still possible to attach an XML file to a CLAW object as a Media (see Datastreams), however there is no mechanism in CLAW
-    for editing XML in a user-friendly way.
\ No newline at end of file
+![Screenshot of the Geographic Locations vocabulary showing example terms in a hierarchy.](../assets/metadata_geographic_location_list.png)
+-->
+
+## Field Types
+
+Each field in Drupal has a _type_ that defines its _properties_ and behavior. These field types also have _widgets_ and _formatters_ that control entry forms and display, respectively.
```

Comment:
I would mention the basic field types here.  Don't have to list them all, but at least mention text and numbers.  Then I'd make h3 headrs (###) for the EDTF, Entity Reference, and Typed Relations sections.